doc/ref/meataxe.xml

```diff
@@ -77,6 +77,16 @@ bases used are needed as well.
 <Section Label="Module Constructions">
 <Heading>Module Constructions</Heading>
 
+<ManSection>
+<Heading>NaturalGModule</Heading>
+<Func Name="NaturalGModule" Arg='group[, field]' Label="for matrix group and a field"/>
+<Description>
+creates a MeatAxe module over <A>field</A> from the generators of the matrix
+group <A>group</A>. If <A>field</A> is not provided then the value returned by
+<Ref Filt="DefaultFieldOfMatrixGroup"/> is used instead.
```

Comment:
```suggestion
<Ref Attr="DefaultFieldOfMatrixGroup"/> is used instead.
```
This is the reason why the test concerning the documentation failed.

I think the problem is that the return value computed in the input file `doc/ref/testconsistency.g` is taken seriously. Part of the code in this file is quite dubious, many of the printed suggestions are simply wrong --luckily nobody has ever tried to follow these suggestions.